Feature/content warning guidelines page

MatthewShields opened the pull request.

MatthewShields committed:
Merge pull request #1 from tatianamac/master

Update master branch from original repo into fork

MatthewShields committed:
Add empty page content file

MatthewShields committed:
Add link in footer

MatthewShields committed:
Make footer links span two columns when in grid template layout

MatthewShields committed:
wrap footer grid-column in media query to strip triggering 2 columns on mobile

MatthewShields committed:
Add guidelines content

MatthewShields committed:
Capitalise 'Warning'

MatthewShields committed:
Add link to guidance page from guidance definition block

MatthewShields committed:
Change 'term' to 'topic'

MatthewShields committed:
Merge branch 'prod' into feature/content-warning-guidelines-page

tatianamac committed:
Merge branch 'prod' into feature/content-warning-guidelines-page

ovlb commented:
@MatthewShields Is this work in progress?

tatianamac commented on 11ty/content-warning-guidelines/index.md:
```diff
@@ -0,0 +1,53 @@
+
+## What is the difference between a Content Warning and a Trigger Warning
+
+You may well have heard these two different phrases used to warn of different topics, but what is the differences? It is sometimes said that content warnings are used to help people avoid topics that may cause offence, upset or anxiety, and that trigger warnings are used when warn of a topic that may cause a traumatic reaction.
```
```suggestion
Content warnings and trigger warnings are sometimes used interchangeably to warn the reader/user when a topic that may cause an undesirable feeling or reaction. We opt to use content warning as trigger warning is more specific (and makes the assumption that the reader/user will be triggered).
```

tatianamac commented on 11ty/content-warning-guidelines/index.md:
```diff
@@ -0,0 +1,53 @@
+
+You may well have heard these two different phrases used to warn of different topics, but what is the differences? It is sometimes said that content warnings are used to help people avoid topics that may cause offence, upset or anxiety, and that trigger warnings are used when warn of a topic that may cause a traumatic reaction.
+
+This however is very hard to gauge. People can have very different reactions, just as they may have very different personal experiences, with any topic that you may discuss. Their form follows the same rules though and so for this reason we will be using the term Content Warning for these guidelines.
```
```suggestion

```

tatianamac commented on 11ty/content-warning-guidelines/index.md:
```diff
@@ -0,0 +1,53 @@
+
+This however is very hard to gauge. People can have very different reactions, just as they may have very different personal experiences, with any topic that you may discuss. Their form follows the same rules though and so for this reason we will be using the term Content Warning for these guidelines.
+
+The most important thing though, is that a warning is given, rather than which term is used.
```
```suggestion

```

tatianamac replied:
I get why you explained this but I think it suffices to say the difference and why we chose one over the other.

tatianamac commented on 11ty/content-warning-guidelines/index.md:
```diff
@@ -0,0 +1,53 @@
+  <li>CW: [topic or topics]</li>
+</ul>
+
+We would recommend that wherever possible you choose the first full text example. This will ensure that people who may not be familiar with the acronym of CW will still fully understand. In the case of media such as video, it is vital that it is given in a way that everyone will be able to access it, including through audio or visual only.
```
```suggestion
We would recommend that wherever possible you choose the first full text example. This will ensure that people who may not be familiar with the acronym of CW will still fully understand. In the case of media such as video, it is vital that it is given in a way that everyone will be able to access it, including through audio and visual cues.
```

tatianamac commented on 11ty/content-warning-guidelines/index.md:
```diff
@@ -0,0 +1,53 @@
+
+## Responding to feedback
+
+Even taking this on board, someone may feed back to you saying that your content caused them a negative reaction. If this happens, please wherever possible amend your content warning. This will be appreciated by any others in the future who may also have had this negative reaction without warning.
```
```suggestion
Even taking this on board, someone may feed back to you saying that your content caused them a negative reaction. If this happens, please wherever possible amend your content warning as possible. The goal of content warnings is to reduce as much harm as possible, and it's never too late to start.
```

tatianamac reviewed (approved):
Left a few prescriptive comments!

tatianamac commented:
Thank you, @MatthewShields , for your patience with me on this! I left a few edits to make it more concise.

I believe you'll need to update your branch to resolve the conflicts since we merged a few things in since. Excited to get this up!

MatthewShields commented:
Thanks for the edits Tatiana, I'm glad that on the whole I've managed to convey an idea that you agree with 🙂

I will get these actioned tomorrow as going to head to sleep earlier tonight to take care of myself but it's really much appreciated!

tatianamac commented:
@MatthewShields No rush! Have a good evening. 🌓

MatthewShields committed:
Update 11ty/content-warning-guidelines/index.md

Co-authored-by: Tatiana Mac <github@tatianamac.com>

MatthewShields committed:
Update 11ty/content-warning-guidelines/index.md

Co-authored-by: Tatiana Mac <github@tatianamac.com>

MatthewShields committed:
Update 11ty/content-warning-guidelines/index.md

Co-authored-by: Tatiana Mac <github@tatianamac.com>

MatthewShields committed:
Update 11ty/content-warning-guidelines/index.md

Co-authored-by: Tatiana Mac <github@tatianamac.com>

MatthewShields committed:
Update 11ty/content-warning-guidelines/index.md

Co-authored-by: Tatiana Mac <github@tatianamac.com>

MatthewShields committed:
Merge branch 'prod' into feature/content-warning-guidelines-page

MatthewShields committed:
remove additional line breaks to pass lint tests

MatthewShields commented:
@tatianamac these have all been actioned now and merged, just needed to remove a couple of line breaks to pass the lint testing in the markdown :)

MatthewShields committed:
remove my now unneeded media query

tatianamac commented:
Hi @MatthewShields ! I think this looks good but the longer explanation within each definition is still showing up in the deploy preview. I think we can delete that now.

MatthewShields commented:
Oh apologies I completely missed that, will remove that and link to this page from the top section instead 🙂

MatthewShields committed:
Merge branch 'prod' into feature/content-warning-guidelines-page

MatthewShields committed:
Remove longer content warning from definition page

MatthewShields committed:
Change content warning flag to link to separate content warning guidance page

MatthewShields commented:
@tatianamac - I've updated this PR now to remove the longer content warning flag from the definition page and to link the top version to the separate page.

I've also changed where I had two different sentences for avoid and warning levels to both use this sentence for consistency:

_We would recommend adding a content warning when speaking about this term. Please <a href="/content-warning-guidelines/">read the guidance</a> on how and when to warn people before using this term in any context._

MatthewShields committed:
Merge branch 'prod' into feature/content-warning-guidelines-page

ovlb commented:
@MatthewShields Were there styles you’d written for the larger box that might be redundant now?

MatthewShields committed:
remove unneeded content flag styling

MatthewShields commented:
@ovlb A good point, I thought I remembered them using the same styles as the top section but I just checked and they weren't - have removed now.

Thanks for making me keep it tidy! :)

ovlb commented:
@MatthewShields Thanks for having a look. :)

tatianamac committed:
Merge branch 'prod' into feature/content-warning-guidelines-page
